static/app/views/performance/browser/webVitals/utils/calculatePerformanceScore.tsx

```diff
@@ -32,40 +32,53 @@ export type ProjectScore = {
 };
 
 type Vitals = {
-  cls: number;
-  fcp: number;
-  fid: number;
-  lcp: number;
-  ttfb: number;
+  cls: number | null;
+  fcp: number | null;
+  fid: number | null;
+  lcp: number | null;
+  ttfb: number | null;
 };
 
 export const calculatePerformanceScore = (vitals: Vitals): ProjectScore => {
-  const {cls, fcp, fid, lcp, ttfb} = vitals;
+  const [lcpScore, fcpScore, ttfbScore, clsScore, fidScore] = [
+    'lcp',
+    'fcp',
+    'ttfb',
+    'cls',
+    'fid',
+  ].map(vital => {
+    if (vitals[vital] === null) {
+      return null;
```

Comment:
can we return 0 here and avoid the `?? 0` checks below? Would be nice to avoid setting null whenever possible.